kibana/docker/templates/Dockerfile.j2

```diff
@@ -23,6 +23,11 @@ FROM centos:7
 
 ENV ELASTIC_CONTAINER true
 
+ARG DEFAULT_UID=1000
+ARG DEFAULT_GID=1000
+ENV DEFAULT_UID $DEFAULT_UID
+ENV DEFAULT_GID $DEFAULT_GID
+
```

Comment:
The Kibana dockerfile template also hard-codes UID 1000, although it doesn't have a similar docker entrypoint shell script like Elasticsearch. However I am defining the ENV/ARG values so that it could be overriden at build-time if desired.